src/grafana.js

```diff
@@ -91,8 +91,8 @@ module.exports = (robot) => {
     let apiPanelId = false;
     let pname = false;
     const query = {
-      width: process.env.HUBOT_GRAFANA_DEFAULT_WIDTH || 1000,
-      height: process.env.HUBOT_GRAFANA_DEFAULT_HEIGHT || 500,
+      width: parseInt(process.env.HUBOT_GRAFANA_DEFAULT_WIDTH) || 1000,
+      height: parseInt(process.env.HUBOT_GRAFANA_DEFAULT_HEIGHT) || 500,
```

Comment:
nit: My linter will complain about `parseInt()` if the second `radix` [parameter isn't defined](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#description).

```suggestion
      width: parseInt(process.env.HUBOT_GRAFANA_DEFAULT_WIDTH, 10) || 1000,
      height: parseInt(process.env.HUBOT_GRAFANA_DEFAULT_HEIGHT, 10) || 500,
```